vsm

```diff
@@ -164,8 +193,12 @@ def run(state):
         else:
             process(state, line, None)
 
+def get_runtime():
+    return round(time.time() * 1000 - start_time_ms)
```

Comment:
This looks pretty straight-forward. However, how about using something from the std library such as time.monotonic()? Or, perhaps even better, time.perf_counter()? Given the goal of performance testing these will eventually have and the very short time for processing that we'll have in most cases, maybe perf_counter would be the best. We don't need to start from 0 necessarily, and we can easily calculate 'elapsed time' by saving the first result.